backend/app/api/routes/integrations/home_assistant.py

```
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, Depends
+
+from app.services.today_service import TodayService
+
+router = APIRouter(prefix="/integrations/home-assistant", tags=["integrations"])
+
+
+@router.get("/summary")
+def home_assistant_summary(service: TodayService = Depends(TodayService)) -> dict[str, str | int | None]:
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)